.github/workflows/backend-ncnn.yml

```diff
@@ -18,7 +18,7 @@ concurrency:
 
 jobs:
   test_onnx2ncnn:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
The 'gcc-multilib' step seems not necessary
```
- name: gcc-multilib
        run: sudo apt install gcc-multilib g++-multilib wget libprotobuf-dev protobuf-compiler
```

Reply:
maybe we could optimize the workflow in next PR